CHANGELOG.md

```diff
@@ -2,3 +2,15 @@
 All notable changes to this project will be documented in this file.
 
 Preferably use **Added**, **Changed**, **Removed** and **Fixed** topics in each release or unreleased log for a better organization.
+
+## [1.0.0](https://github.com/quintoandar/hive-metastore-client/releases/tag/1.0.0)
+First modules and entities of Hive Metastore Client package.
+
+### Added
+* Adding thrift files in project ([#4](https://github.com/quintoandar/hive-metastore-client/pull/4))
+* Adding thrift python files ([#9](https://github.com/quintoandar/hive-metastore-client/pull/9))
+* Create clients main class ([#16](https://github.com/quintoandar/hive-metastore-client/pull/16) and [#10](https://github.com/quintoandar/hive-metastore-client/pull/10))
+* Adding DatabaseBuilder ([#13](https://github.com/quintoandar/hive-metastore-client/pull/13))
+* Adding TableBuilder, ColumnBuilder, SerDeInfoBuilder and StorageDescriptorBuilder ([#15](https://github.com/quintoandar/hive-metastore-client/pull/15))
+* Adding PartitionBuilder ([#17](https://github.com/quintoandar/hive-metastore-client/pull/17))
```

Comment:
I also added the method `add_partitions_to_table` we could add here in the change-log :)

Reply:
sure. You can add it if you want. Else I add it later. Tks